astore/client/commands/commands.go

```diff
@@ -33,7 +33,7 @@ func New(base *client.BaseFlags) *Root {
 	root.AddCommand(NewTag(root).Command)
 	root.AddCommand(NewNote(root).Command)
 	root.AddCommand(NewPublic(root).Command)
-
+	root.AddCommand(NewDelete(root).Command)
```

Comment:
Up to you, but I'm personally a fan of continuous, small incremental updates on master. So the code gets tested by automated pipelines (well, once we have them...), other folks in the team have visibility into the features, and there are fewer merge conflicts as we go on, with less branching and merging.

If you remove the registration of the sub-command - so effectively the code is not released/disabled - and get the code to compile and `go fmt` clean, I'd be happy to have this merged already.